Turf/Turf.swift

```diff
@@ -285,3 +288,67 @@ public struct Polyline {
         return closestCoordinate
     }
 }
+
+public struct Polygon {
+    var polygonCoordinates: [[CLLocationCoordinate2D]]
+    
+    var polygonArea: Double {
```

Comment:
This property already lives inside a struct named `Polygon`, so the word “polygon” here is [redundant](https://swift.org/documentation/api-design-guidelines/#omit-needless-words).